scripts/cxg_remaster.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import time
+
+import numpy as np
+import psutil
```

Comment:
I'm not sure what subsystem will run this script, but `psutil` is not in any of the requirements.txt module dependency files.  Is it needed for deployment?

Reply:
Not needed for deployment. Will add a note in the script that this is a requirement not mentioned in the top level file